frontend/src/components/SeedlotRegistrationSteps/OrchardStep/data.ts

```diff
@@ -0,0 +1,13 @@
+const FemaleGameticOptions = [
+  'F1 - Visual estimate',
+  'F2 - Measured cone volume',
+  'F3 - Cone weight',
+  'F4 - Cone number from weight',
+  'F5 - Cone number from standard volume',
+  'F6 - Sample of seeds',
+  'F7 - Filled Seeds',
+  'F8 - Ramet proportion by clone',
+  'F9 - Ramet proportion by age and expected production'
+];
+
+export default FemaleGameticOptions;
```

Comment:
We already have these values on the backend. What do you think of updating to retrieve from there?

Reply:
Not sure if that is necessary, since those values will be static. I think we can leave as it is and maybe re-structure (if necessary) when we connect with the real API.

Reply:
Got it. For now, there are other features that should be focused on. You have my blessing for this one 💯